bundles/org.openhab.binding.phc/src/main/java/org/openhab/binding/phc/internal/util/StringUtils.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.phc.internal.util;
+
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link StringUtils} class defines some static string utility methods
+ *
+ * @author Leo Siepel - Initial contribution
+ */
+public class StringUtils {
```

Comment:
Can you please add `@NonNullByDefault`

Reply:
Now I see after merging that it was also missing in the new classes you created for the sonos and upnpcontrol bindings.
If you could add them, that would be better.

Reply:
Yes, done. Will do the others too.